doc/source/howtos/upgrade_guide.rst

```diff
@@ -4,14 +4,16 @@
 Upgrading Percona XtraDB Cluster
 ================================
 
-This guide describes the procedure for upgrading |PXC| version 5.6
-to the latest version 5.7 without downtime (*rolling upgrade*).
-This means that replication between nodes will continue during the upgrade.
-
-Rolling upgrade from earlier versions to 5.7 is not supported.
-If you are running |PXC| version 5.5, it is recommended to shut down all nodes,
-then remove and re-create the cluster from scratch.
-Alternatively, you can perform a `rolling upgrade from PXC 5.5 to 5.6 <https://www.percona.com/doc/percona-xtradb-cluster/5.6/upgrading_guide_55_56.html>`_,
+This guide describes the procedure for upgrading |PXC| without downtime
+(*rolling upgrade*) to the latest 5.7 version. Rolling upgrade means that
+replication between nodes will continue during the upgrade.
+
+As major upgrade (from 5.6 to 5.7 version), so minor one (from 5.7.x
+to 5.7.y)  can be done in this way. Rolling upgrade to 5.7 from versions
```

Comment:
The first sentence seems to have grammar issues - "As major upgrade (from 5.6 to 5.7 version)," - the meaning is not defined/stated?

Reply:
Hm... It's a typical construction, "as ... so", e.g. "as IST, so SST may occur". I'll change it to "both ... and ..." for better readability.